platform/commonUI/edit/src/actions/SaveAction.js

```diff
@@ -19,14 +19,12 @@
  * this source code distribution or the Licensing information page available
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
-/*global define*/
 /*jslint es5: true */
```

Comment:
I added this jslint rule in a couple of places. It's now redundant (because we're using jshint 2+ which [enables ES5 mode by default](https://jslinterrors.com/es5-option-is-now-set-per-default) ). This rule was added specifically because jslint did not like the reserved work 'catch' being used as a function name.